### DISCUSSION TITLE
Support for Trans component

### DISCUSSION THREAD

**Discussion by Pardaramskha** [i18next/next-i18next#1594] [Q&A] [answered]
Hello,
Is there any support for react-i18next Trans component ? Or another way to include tags within translated content ?

Best regards,

**Comment by isaachinman** [comment, marked as answer]
Yes, `next-i18next` inserts an [`I18nextProvider`](https://github.com/isaachinman/next-i18next/blob/master/src/appWithTranslation.tsx#L66), so you can use any i18next components or utils that you'd like to.
